mypy.ini

```diff
@@ -0,0 +1,3 @@
+[mypy]
+python_version = 3.6
```

Comment:
Aw hey just noticed this line while doing a final check. Do we need it? We removed it from the analogous certbot file in https://github.com/certbot/certbot/pull/8426.